client/components/paths/path-single.js

```diff
@@ -138,68 +121,27 @@ class SinglePath extends Component {
         { pathSteps[0].step !== null &&
           <PathProgress progress={this.getCompletePercentage()} />
         }
-
         <div style={styles.container}>
           <List>
             { pathSteps[0].step !== null &&
               pathSteps.map(step => {
                 const stepUrl = step.resource.properties.url
                 const resourceImg = step.resource.properties.imageUrl
                 return (
-                <div key={stepUrl}>
-                  <ListItem
-                    key={stepUrl}
-                    role={undefined}
-                    dense
-                    button
-                    disableRipple
-                  >
-                    <Checkbox
-                      onChange={this.handleCompletedClick(stepUrl)}
-                      checked={this.checkForComplete(stepUrl)}
-                      disableRipple
-                    />
-
-                    {
-                      <img src={resourceImg ? resourceImg : "../../default.png"} width={75} />
-                    }
-
-                    <ListItemText primary={step.resource.properties.name} />
-
-                    {this.state.selectedItems.indexOf(stepUrl) !== -1 ?
-                      <ExpandLess
-                        onClick={() => this.handleDropdownClick(stepUrl)}
-                      /> :
-                      <ExpandMore
-                        onClick={() => this.handleCollapseClick(stepUrl)}
-                      />
-                    }
-
-                  </ListItem>
-
-                  <Collapse
-                    in={this.state.selectedItems.indexOf(stepUrl) !== -1}
-                    timeout="auto" unmountOnExit>
-                    <List component="div" disablePadding>
-                      <ListItem button>
-                        <ul>
-                          <li>Description: {step.resource.properties.description}</li>
-                          <li>Visit resource: <a href={step.resource.properties.url} target="_blank">{step.resource.properties.name}</a></li>
-                        </ul>
-                      </ListItem>
-                    </List>
-                  </Collapse>
-
-                  </div>
+                  <ResourceCard 
```

Comment:
I think I remember us saying we didn't want full reviews on the dashboard, only the # of stars.  Maybe we can discuss again as a group?

Reply:
I'll leave the reviews in here for this pull request, but I'll remove it in the future.  Right now I'm only pulling the top three reviews.  We just need to discuss what we're doing with them, wether they show in its own page or something.